deps/actual_build.jl

```diff
@@ -0,0 +1,102 @@
+using BinDeps
+using BinDeps: builddir
+
+BinDeps.@setup
+
+const FFTW_VER = v"3.3.6-pl2"
+
+if Sys.iswindows()
+    const libfftw_name = "libfftw3"
+    const libfftwf_name = "libfftw3f"
+else
+    const libfftw_name = "libfftw3_threads"
+    const libfftwf_name = "libfftw3f_threads"
+end
+
+# Why can't everyone just agree on what to call this library...
+function makealiases(lib)
+    major = string(FFTW_VER.major)
+    nover = replace(lib, major, "")
+    return String[
+        nover,
+        join([lib, Libdl.dlext, major], "."),
+        join([nover, Libdl.dlext, major], "."),
+        lib * "-" * major,
+        nover * "-" * major,
+    ]
+end
+
+libfftw = library_dependency(libfftw_name, aliases=makealiases(libfftw_name))
+libfftwf = library_dependency(libfftwf_name, aliases=makealiases(libfftwf_name))
+
+isdir(joinpath(@__DIR__, "usr", "lib")) || mkpath(joinpath(@__DIR__, "usr", "lib"))
+
+const URL = "https://github.com/ararslan/fftw-builder/releases/download/v$FFTW_VER/libfftw-$FFTW_VER"
+
+# Mapping of (Sys.KERNEL, Sys.ARCH) to (url, sha) for precompiled binaries from fftw-builder
+const downloads = Dict(
+    (:Linux, :x86_64)   => ("$URL-linux-x86_64.tar.gz",
```

Comment:
This needs to check the full GNU triple of `Sys.MACHINE`, just kernel and arch aren't good enough. If someone is running Julia on a non-glibc Linux distro, these binaries won't work and this should use a source build.